lib/compiler.js

```diff
@@ -65,6 +65,13 @@ exports.compile = function (code, options) {
     });
 
     importExportVisitor.finalizeHoisting();
+
+    if (getOption(options, "wrapWithRun")) {
+      result.code = "module.run(function(){" +
+        magicString.toString() + "//*/\n});";
+
+      return result;
+    }
```

Comment:
>For example, if I flip wrapWithRun to true in the REPL, I can't use any of the symbols I've imported, because the variable declaration ends up inside the module.run callback. 

Ah ok, how does the normal module wrapper not interfere?

> If we had something like options.repl, we might even be able to get rid of options.generateLetDeclarations since the REPL is the only consumer that really needs to generate var declarations (unless I'm mistaken).

Oh niice! `options.repl` is nice and clean. We also have a `utils.isREPL()` helper to auto detect.